fili-core/src/main/java/com/yahoo/bard/webservice/data/HttpResponseMaker.java

```diff
@@ -136,7 +136,7 @@ private ResponseBuilder createResponseBuilder(
                                 LinkedHashMap::new
                         ));
 
-        Response response = new Response(
+        ResponseData response = new ResponseData(
```

Comment:
@archolewa I have trouble making `ResponseWriter` injectable. 

The default `ResponseWriter` requires `ResponseWriterSelector` which requires all three pre-defined `ResponseWriter` to be instantiated  (JSON, JsonApi and Csv) at binding time. But `JsonResponseWriter` and `JsonApiResponseWriter` needs two variables: `responseData` and `paginationLinks` that are not available at binding time.  Is there a way to work around this.